Update the specs for re-relaying (#1)
orta/relay: website/spec/Connections.md

GraphQL Cursor Connections Specification
--------------------------------------

> This specification aims to provide options for GraphQL clients to consistently 

Reviewer: ```suggestion
This specification aims to provide an option for GraphQL clients to consistently 
```

handle pagination with support for related metadata via a GraphQL server. This

> spec proposes calling them connections and treats pageable list in a standardized 

Reviewer: ```suggestion
spec proposes calling them connections and exposing them in a standardized 
```

way. 

In the query, the connection model provides a standard mechanism for slicing 

> and paginating the result set.

Reviewer: I have a hard time understanding this sentence. I know you didn’t write this, but do you understand what it’s trying to convey?

Author: I think it's talking about always using the same args in each connection for pagination

Author: e.g. `before` `after` `next` `the_other_one`

Reviewer: After reading further, I think they mean in the literal query _text_, and then what you said yeah

Whatever type this field returns will be referred to as the *cursor type*
in the rest of this spec.

> The result of this field is considered opaque by the spec, but will be passed

Reviewer: I wonder if this could also be intended as:

```suggestion
The result of this field should be considered opaque by the client, but will be passed
```

🤔 Not sure

Author: I've not got a strong opinion either way on this

Reviewer: The point is often made [or I'm just making this up] that clients should treat these cursors opaque, hence my thought that maybe by “Relay” they really meant “client”.

`startCursor` and `endCursor` must be the cursors corresponding to the first and
last nodes in `edges`, respectively.

> NOTE As this spec was created with Relay in mind, it's worth noting that 

Reviewer: ```suggestion
NOTE As this spec was originally created with Relay Classic in mind, it's worth noting that 
```
